### PR TITLE
Kimchi

### PR DESCRIPTION
**Pull request by GabrielSac**
Add kimchi verifier.
Change module name from verification to verify

### CODE REVIEW

File: verifiers/kimchi/to_base64/proof_to_base64.go
Comment (JulianGCalderon) [inline verifiers/kimchi/to_base64/proof_to_base64.go:1]: We can remove this file, and use base64 command

File: verifiers/kimchi/lib/libkimchi_verifier.a
Comment (JulianGCalderon) [inline verifiers/kimchi/lib/libkimchi_verifier.a:1]: We should remove this file and the .dylib

File: verifiers/kimchi/lib/kimchi_verifier_index.bin
Comment (JulianGCalderon) [inline verifiers/kimchi/lib/kimchi_verifier_index.bin:1]: Remove this file

File: verifiers/kimchi/lib/kimchi_ec_add.proof
Comment (JulianGCalderon) [inline verifiers/kimchi/lib/kimchi_ec_add.proof:1]: We can put the proof in the proover_examples directory

File: verifiers/kimchi/example/kimchi_ec_add.proof.example
Comment (JulianGCalderon) [inline verifiers/kimchi/example/kimchi_ec_add.proof.example:1]: We should move this examples to proover_examples

File: verifiers/kimchi/Makefile
Comment (JulianGCalderon) [inline verifiers/kimchi/Makefile:1]: Trailing newline is missing

**Review by JulianGCalderon — CHANGES_REQUESTED** [review]

File: verifiers/kimchi/lib/src/ec_add_proof_generator/main.rs
Comment (JulianGCalderon) [inline verifiers/kimchi/lib/src/ec_add_proof_generator/main.rs:1]: This main should go to prover examples, as it constructs a proof.

**Review by JulianGCalderon — CHANGES_REQUESTED** [review]

**Review by JulianGCalderon — APPROVED** [review]
LGTM

**Review by jotabulacios — APPROVED** [review]
LGTM

**Pull request merged**